### PR TITLE
feat: create split board screen - board name input and tip bar

### LINKED ISSUE #137: [FEATURE]: Create split retro board
## Summary

As a user I want to create a main board and split my team users into multiple sub teams. I also want to be able to customize some board configurations

### Basic example

image

### PR DESCRIPTION
**Pull request by nunocaseiro**
Relates to #137 

## Screenshots

image

## Proposed Changes

  - Create board screen
  - Create board bar tips
  - Create board content with the board name input
  
  This pull request is intended to close #137

### CODE REVIEW

File: frontend/components/CreateBoard/Content/Content.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    setOpened(false);
+  };
+
+  return (
+    <StyledForm
+      direction="column"
+      css={{ width: "100%", height: "100%", backgroundColor: "$background" }}
+      onSubmit={methods.handleSubmit(() => {
+        // saveBoard(text, maxVotes);
+      })}
```
Comment (RubenMCCarreira) [inline frontend/components/CreateBoard/Content/Content.tsx:45]: improv this pls

File: frontend/components/CreateBoard/Content/Content.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+          height: "100%",
+          pt: "$64",
+          pl: "$152",
+          pr: "$92",
+          overflow: "auto",
+        }}
+      >
+        <FormProvider {...methods}>
+          <BoardName mainBoardName={mainBoardName} />
+          {/* <Settings /> */}
```
Comment (RubenMCCarreira) [inline frontend/components/CreateBoard/Content/Content.tsx:60]: comment here, pls remove

File: frontend/components/icons/Info.tsx [hunk 3]
```diff
@@ -1,15 +1,62 @@
...
+          fill="currentColor"
+        />
```
Comment (RubenMCCarreira) [inline frontend/components/icons/Info.tsx:29]: can you extract this path to other file? 
the same to next paths too, pls

**Review by RubenMCCarreira — APPROVED** [review]

File: frontend/components/CreateBoard/Content/Content.tsx (hunk 1 above)
Comment (nunocaseiro) [reply frontend/components/CreateBoard/Content/Content.tsx:45]: this is for the next feature

**Review by RubenMCCarreira — APPROVED** [review]

**Pull request merged**